metricbeat/module/apache/status/status.go

```diff
@@ -87,10 +92,36 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	}
 
 	data, _ := eventMapping(scanner, m.Host())
+	event := mb.Event{
+		MetricSetFields: data,
+	}
 
-	if reported := reporter.Event(mb.Event{MetricSetFields: data}); !reported {
-		m.Logger().Error("error reporting event")
+	if m.isFleetMode {
+		event = adjustFleetEvent(event)
 	}
 
+	if reported := reporter.Event(event); !reported {
+		m.Logger().Error("error reporting event")
+	}
 	return nil
 }
+
+func adjustFleetEvent(event mb.Event) mb.Event {
+	var adjusted mb.Event
+	adjusted.MetricSetFields = event.MetricSetFields.Clone()
+
+	// Remove apache.hostname
+	adjusted.MetricSetFields.Delete("hostname")
+
+	// Replace "apache.uptime" with "uptime"
+	uptime, err := adjusted.MetricSetFields.GetValue("uptime.uptime")
+	if err == nil {
+		if adjusted.RootFields == nil {
+			adjusted.RootFields = common.MapStr{}
+		}
+
+		adjusted.RootFields.Put("host.uptime", uptime)
```

Comment:
I think that the uptime reported by apache is the time since the apache server was started, not the host uptime, so probably they are fine where they are, or in any case they should be moved to `process.uptime`, or to some field for service uptimes.

```
root@8673a9a186a2:/usr/local/apache2# curl -s localhost:80/server-status?auto | grep Uptime
ServerUptimeSeconds: 59
ServerUptime: 59 seconds
Uptime: 59
root@8673a9a186a2:/usr/local/apache2# uptime
 11:41:49 up 6 days, 17:03,  0 users,  load average: 2.65, 2.96, 2.77
```

Reply:
I'm aware of this. This is an outcome of the migration assessment reviewed by @sorantis. I assumed that it's maybe because of the interpretation of the "host". Anyway, I'm fine to either switch it to the `process.uptime` or leave it as is. WDYT?

Reply:
`process.uptime` sounds good to me for this value.

Reply:
But actually leaving it as is also sounds good to me :slightly_smiling_face:

Reply:
Ok then, let's don't complicate the source. I'm leaving the uptime as it. Thank you for the discussion!